docs/development-guide/IntegrationTests.md

```diff
@@ -21,6 +21,10 @@ The Aurora integration tests are focused on testing connection strings and failo
 The tests are run in Docker but make a connection to test against an Aurora cluster.
 PostgreSQL and MySQL tests are currently supported.
 
+#### Supported Aurora PostgreSQL Versions
```

Comment:
Could we add this to the main README? (https://github.com/awslabs/aws-advanced-jdbc-wrapper#aurora-engine-version-testing)

Reply:
For sure, I entirely missed that table